src/views/UserPanelView/Components/UserGroupTabs.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Tab } from '@material-ui/core'
+import { TabContext, TabList, TabPanel } from '@material-ui/lab'
+import { UsersPanelTable } from './UsersTable'
+
+const GROUPROLES = {
+  admins: { value: 'admin', label: 'Administrators' },
+  customers: { value: 'customer', label: 'Customers' }
+}
+
+export const UserGroupTabs = ({ classes }) => {
+  const [value, setValue] = React.useState(GROUPROLES.admins.value)
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue)
+  }
+  return (
+    <div>
```

Comment:
If we are using mui components we should use Box or fragment <> </>, in this case you can use the react fragment because you don't require css